src/components/routeComponents/markAttendance/MarkAttendance.jsx

```diff
@@ -334,19 +335,23 @@ class MarkAttendance extends React.Component {
 	/**
 	 * Method set the coordinator login credential
 	 */
-	setCoordinatorLogin = () => {
+	setCoordinatorLogin = async () => {
 		const { INVALID_CREDENTIALS } = this.props.constants;
 		const { hasError, formData } = this.state;
-		const { appConfig, coordinatorLoginSuccess } = this.props;
-		const { coordinatorCredentials } = appConfig;
+		const { coordinatorLoginSuccess } = this.props;
+
+		const coordinatorList = await fetchCoordinator({ secretKey: '451725' });
```

Comment:
This API will return you a single object so no need to map on the list.
```suggestion
		const fetchedCoordinator = await fetchCoordinator({ secretKey, whatsappNumber });
```